dist/index.js

```diff
@@ -19,10 +19,30 @@ const CONFIG = {
         config: {
             name: 'Config',
             fields: {
-                projectId: {
+                vertexAiProjectId: {
                     row: 1,
                     col: 2,
                 },
+                vertexAiLocation: {
+                    row: 2,
+                    col: 2,
+                },
+                vertexAiEndpoint: {
```

Comment:
This doesn't need to be a user config; it's sufficient to have it in the app config under `vertexAi`